### PR TITLE
Release/37

### PR DESCRIPTION
**Pull request by renatoAnyline**

### CODE REVIEW

File: plugin/plugin.xml [hunk 1]
```diff
@@ -44,7 +44,7 @@
         <!-- <resource-file src="src/ios/anyline_sdk/AnylineResources.bundle"/>-->
 
         <!-- Anyline Framework & Resource Bundle CocoaPods -->
-        <framework src="Anyline" type="podspec" swift-version="4.2" spec="~> 35"/>
+        <framework src="Anyline" type="podspec" swift-version="4.2" spec="37"/>
```
Comment (philippAnyline) [inline plugin/plugin.xml:47]: should we keep hotfix / minor release included or not?

Comment (philippAnyline) [reply plugin/plugin.xml:47]: ~> will include every version from 37.0.0 to 38.0.0 (not including 38.0.0)

File: plugin/src/ios/ALPluginHelper.m [hunk 2]
```diff
@@ -617,6 +617,23 @@ + (NSDictionary *)dictionaryForCompositeResult:(ALCompositeResult *)scanResult
...
+    NSMutableDictionary *dictResult = [NSMutableDictionary dictionaryWithCapacity:4];
+    
+    [dictResult setObject:scanResult.result forKey:@"text"];
+    
+    NSString *imagePath = [ALPluginHelper saveImageToFileSystem:scanResult.image compressionQuality:dividedCompRate];
+    
+    [dictResult setValue:imagePath forKey:@"imagePath"];
+    
+    NSString *fullImagePath = [ALPluginHelper saveImageToFileSystem:scanResult.fullImage compressionQuality:dividedCompRate];
+    [dictResult setValue:fullImagePath forKey:@"fullImagePath"];
```
Comment (philippAnyline) [inline plugin/src/ios/ALPluginHelper.m:633]: We do not add pluginID and confidences

File: plugin/plugin.xml (hunk 1 above)
Comment (renatoAnyline) [reply plugin/plugin.xml:47]: I have no position on that so I will add the ~> back

**Pull request merged**